addons/test_website/tests/test_controller_args.py

```diff
@@ -29,3 +30,25 @@ def test_crawl_args(self):
         req = self.url_open('/ignore_args/kw?a=valueA&b=valueB')
         self.assertEqual(req.status_code, 200)
         self.assertEqual(req.json(), {'a': 'valueA', 'kw': {'b': 'valueB'}})
+
+    def test_route_type(self):
```

Comment:
Maybe rename to test_route_type_all and add a bit of docstring?

Reply:
Also, side note for commit message:
"Actually" is a fake friend in English, its meaning is close to "En fait" in French.
If you want "Actuellement", "Currently" is more suitable.